package.json

```diff
@@ -78,5 +77,5 @@
     "tiny-glob": "^0.2.9",
     "typescript": "^4.9.5"
   },
-  "packageManager": "pnpm@7.29.0"
+  "packageManager": "pnpm@8.6.0"
```

Comment:
The latest version is `8.6.12`, although I'd consider future-proofing this to just `8.6` or even `8`. pnpm has a good major stability track record from my experience

Reply:
Yep that's a great call, this was a byproduct of me fighting with Tailwind config not upgrading properly 🙃 

I'll set it to `8`, agreed pnpm is good about semver releases